sdk/deviceprovisioningservices/azure-resourcemanager-deviceprovisioningservices/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0 (2021-05-28)
+
+- Azure Resource Manager IotDps client library for Java. This package contains Microsoft Azure SDK for IotDps Management SDK. API for using the Azure IoT Hub Device Provisioning Service features. Package tag package-2020-03. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
Why is IoT and Dps together with no space? Why is t in IoT and ps in Dps lowercase?

Reply:
This was all autogenerated, but I can spell out the acronyms here, sure

Reply:
autogenerated from what? swagger?

Reply:
Service name here as `IotDps` can be customized in readme.java.md, add a `service-name: <preferred name>` would be fine.